### DISCUSSION TITLE
Difficulty slider in the beatmap search

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#17339] [Ideas]
A slider of difficulty on the beatmap listing could be useful instead of having to write _star<7.4 star>4.5_

![image](https://user-images.githubusercontent.com/40531871/159101747-935342ff-2a9d-4863-9531-3994ead4cb4e.png)

**Comment by godlikeswan** [comment]
Should be marked as "answered"/implemented?

**Reply by Joehuu** [reply]
It's only in song select right now, not in the online beatmap listing.
